core/libp2p/peer/protocol.hpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_PROTOCOL_HPP
+#define KAGOME_PROTOCOL_HPP
+
+#include <string>
+
+namespace libp2p::peer {
+
+  using Protocol = std::string;
```

Comment:
Can multiaddr protocols be used here?

Reply:
No. Example of Protocol is `/bittorrent/1.0` or `/ping/1.0.0`